pkgs/shells/fish/plugins/forgit.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildFishPlugin, fetchFromGitHub, git, fzf }:
+
+buildFishPlugin rec {
+  pname = "forgit";
+  version = "unstable-2021-04-09";
+
+  buildInputs = [ git fzf ];
```

Comment:
I don't think this has any effect because `git` and `fzf` are runtime dependencies of the plugin, not build dependencies.
You might have to `substituteInPlace` the references to `git`, `fzf` and `bat` with their nix store paths in the plugin's script instead.

Reply:
that was second too late..

Reply:
Got it, thanks for commenting. Let me try tomorrow and make a PR after I've checked.

Reply:
> But aren't nativeBuildInputs the build dependencies?
> Also, I just checked the code, it always refers to them without the full path, so if they're the path it should work.

To clarify: since there's no linking step in `buildFishPlugin`, and since `buildInputs` doesn't alter the user's `PATH`, the dependencies to the nix store paths are lost for runtime.

Reply:
Well you were right!
env: fzf: No such file or directory